Lombiq.Analyzers/Build.props

```diff
@@ -24,6 +22,12 @@
     <_SkipUpgradeNetAnalyzersNuGetWarning>true</_SkipUpgradeNetAnalyzersNuGetWarning>
   </PropertyGroup>
 
+  <!-- Don't run analyzers during Visual Studio build, not to slow down development. They still run during dotnet build.
+       By making this conditional, you can still set the variable to true even when using the NuGet package. -->
+  <PropertyGroup Condition="'$(RunAnalyzersDuringBuild)' != 'true'">
+    <RunAnalyzersDuringBuild>false</RunAnalyzersDuringBuild>
```

Comment:
> By making this conditional, you can still set the variable to true even when using the NuGet package.

Actually, it's different than that. When you reference the NuGet package, the associated _.props_ file is imported from the consuming project before the beginning of its project file, so the condition here will never see the `RunAnalyzersDuringBuild` property being set in the consumer's project file.

But setting it from the _Directory.Build.props_ actually precedes the import of the _.props_ file from the NuGet package, and to respect that, your workaround is actually necessary. 😮